frontend/src/extensions.ts

```diff
@@ -19,6 +67,8 @@ export interface ExtensionModule {
   onPageLoad?: () => void;
   /** Run some code after the page for this extension has loaded. */
   onExtensionPageLoad?: () => void;
+
+  api: ExtensionApi;
```

Comment:
This is in the wrong place IMHO - this interface is for what the extension can provide. Maybe we could pass a "context" object ({ api: ExtensionApi}) to all of the event handlers instead?

Reply:
Yeah I didn't notice the comment above the interface. I think passing a context works, and it's flexible to be expanded in the future